### ISSUE TITLE
Auto Pray Flick Enhancement

### ISSUE THREAD

**Issue by Gittyhub123** [Ganom/Supporter-Issue-Tracker#198]
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
It would be nice if the Auto Pray Flick plugin had an option to work on mouseover. Wouldn't mind if it was part of bronze or plat. In addition to that, it would be nice to be able to determine if the prayer should be on or off when toggling the auto pray flick plugin off. 

Example: You move the mouse to the quick pray orb, and it starts flicking automatically without any other input. You move it away from it (click an npc for instance, or run away from AoE attacks), and it stops auto flicking, then starts again when you move it back to the quick pray orb.

Example for prayers being enabled or disabled when you move the mouse away from the quick pray orb: When the auto pray flick plugin is toggled off by moving the mouse away from the quick pray orb, it would be nice to have the prayer remain on in the instance that you're being attacked but have to move from an AoE attack. It would also be nice for it to turn off if you're looting or running to kill another npc while not being under attack to save prayer. Therefore a toggle would nice preferred.

**Describe alternatives you've considered**
Using a button to toggle it on and off, but it's more of a hassle and sometimes it's easier to play with one hand, and it takes a lot more effort to add timing and focus onto controlling the keyboard as well.

I have not found a way to toggle the prayer being on or off when disabling the auto pray flick plugin.

**Additional context**
N/A

**Comment by Failed4life** [comment]
See Combat Helper

**Comment by Gittyhub123** [comment]
> See Combat Helper

It doesn't do what's described. I appreciate that you've had a look at my suggestion, and maybe it's not worth the time to make at all, but the combat helper, as far as I can tell, only lets you find prayer flicking to a button, it doesn't let you pray flick on mouseover. 

Also, when you deactivate the prayer, it would be nice to be able to toggle if you should keep praying (as it currently works) or if the plugin should disable prayers before stopping the prayer flicking to save prayer points.

**Comment by Failed4life** [comment]
> only lets you find prayer flicking to a button, it doesn't let you pray flick on mouseover.

This is correct, however, as long as you have it toggled on, it'll start pray flicking as soon as you enter combat

**Comment by Gittyhub123** [comment]
> > only lets you find prayer flicking to a button, it doesn't let you pray flick on mouseover.
> 
> This is correct, however, as long as you have it toggled on, it'll start pray flicking as soon as you enter combat

I didn't know. When I tested it I used it to toggle on and off between kills, and it always remained on between kills so it seemed like a slight waste of prayer points. 

So, I'm supposed to toggle it on, have it on one tick flicking, and leave it on between kills - meaning I can click to attack npcs while it 1 tick flicks? I don't mind trying that, but using the auto prayer flicker and toggle it off between kills feel safer - ban wise. Having it as a mouse over would make it more human like I think.
